### ISSUE TITLE
Review: 2.1.2 No Keyboard Trap (A)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#37]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 2.1.2 considering the mobile research questions (to be completed)

---
If keyboard focus can be moved to a component of the page using a keyboard interface, then focus can be moved away from that component using only a keyboard interface, and, if it requires more than unmodified arrow or tab keys or other standard exit methods, the user is advised of the method for moving focus away.

Note 1: Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the Web page (whether it is used to meet other success criteria or not) must meet this success criterion. See Conformance Requirement 5: Non-Interference.
---
Sufficient Techniques for Success Criterion 2.1.2

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    [G21: Ensuring that users are not trapped in content ](https://www.w3.org/WAI/WCAG22/Techniques/general/G21.html)

Failures for Success Criterion 2.1.2

    F10: Failure of Success Criterion 2.1.2 and Conformance Requirement 5 due to combining multiple content formats in a way that traps users inside one format type
